.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: java
 matrix:
   include:
-    - jdk: openjdk8
-      script: ./mvnw install
     - jdk: openjdk11
       # This is the primary target platform. In this build perform a SonarQube
       # analysis.
       script: ./mvnw install sonar:sonar
+    - jdk: openjdk17
```

Comment:
In order to build with JDK 16+ some [changes](https://errorprone.info/docs/installation#jdk-16) to the error-prone profile are required; at least when I `mvn clean verify` Jolo.